{"uuid": "deaab06f-954f-4c40-b11c-107f181309f8", "children": ["4a497780-3e19-4495-b394-d0d9327165a7", "45743cde-52b3-4eb8-ae3e-9658486951b0", "db123b1d-97d1-4a06-b35d-29ceefc06396", "51b776c0-eb20-4646-bbd7-f4bfc829d7f4", "40d78e83-454f-4a4f-80a7-48cc986ded1e", "2e967873-8a3e-4f25-9649-2bc59f343f76"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'wlan0-1'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 408, in get_interface_details\n    wifi_info[i].append(data[i])\n"}, "attachments": [{"name": "ucentral_config: ", "source": "34e396c9-4ebf-4ad5-a211-3869a20bcec0-attachment.attach"}, {"name": "Time Took to apply Config: 39.95749092102051", "source": "2306a29a-424a-469b-8a7e-96eba100f82c-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "9bdc150a-287d-48cd-be85-a037aa42c8dc-attachment.attach"}, {"name": "AP LOgs: ", "source": "21f9eadc-8cb0-41ab-89ac-6d20525a4b99-attachment.attach"}], "start": 1629506097762, "stop": 1629506170204}], "start": 1629506097762, "stop": 1629506170482}